crates/macro/src/derive_props/builder.rs

```diff
@@ -36,12 +35,9 @@ impl ToTokens for PropsBuilder<'_> {
             ..
         } = props;
 
-        let step_trait_repeat = iter::repeat(step_trait);
-        let vis_repeat = iter::repeat(&vis);
-
         let build_step = self.build_step();
-        let impl_steps = self.impl_steps();
-        let set_fields = self.set_fields();
+        let impl_steps = self.impl_steps().into_iter();
+        let set_fields = self.set_fields().into_iter();
```

Comment:
Is this needed?

Reply:
Nope, I thought it did because of `impl_steps` which I also misused. As `set_fields` is a `impl Iter`, it can be used directly.